libcryptsetup-rs-sys/build.rs

```diff
@@ -24,7 +27,16 @@ fn build_safe_free() {
 }
 
 fn generate_bindings(safe_free_is_needed: bool) {
-    let builder = bindgen::Builder::default().header("header.h").size_t_is_usize(true);
+    match pkg_config::probe_library("libcryptsetup") {
+        Ok(l) => {
+            let builder = bindgen::Builder::default()
+                .clang_args(l.include_paths.iter().map(|path| {
+                    let r = format!("-I{}", path.to_string_lossy());
+                    eprintln!("{}", r);
+                    r
```

Comment:
I think this could just be:

```
format!("-I{}", path.to_string_lossy())
```

I don't think `eprintln!()` does anything here.